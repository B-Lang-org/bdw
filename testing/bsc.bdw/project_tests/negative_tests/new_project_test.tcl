
# Creates a new project
handlers::new_project  

# Creates a new project
handlers::new_project project 

# Creates a new project
handlers::new_project project1 -loc dddd

# Creates a new project
handlers::new_project project1 -location dddd

# Creates a new project
handlers::new_project project1 -location mult4 -location mult4 

# Opens the project project.bspec
handlers::open_project project.bspec

# Sets the top file and module 
commands::set_top_module mult4/Mult4Tb.bsv mkMult4Tb

# Creates a new project
handlers::new_project project1 -location mult4 

# Closes the project 
handlers::close_project

# Exits the worksptation
exit

