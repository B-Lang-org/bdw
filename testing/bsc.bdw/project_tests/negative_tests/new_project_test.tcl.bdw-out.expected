Project name is not specified
The project 'project' already exists.
Unknown option: -loc
The current project has not been saved.
The option -location has already been specified
The current project has not been saved.
The current project has not been saved.
The current project has not been saved.
